Dshop/apps/payments/views.py

```diff
@@ -13,3 +15,68 @@ def stripe_config(request):
     if request.method == 'GET':
         stripe_config = {'publicKey': settings.STRIPE_PUBLISHABLE_KEY}
         return JsonResponse(stripe_config, safe=False)
+
+
+@csrf_exempt
+def create_checkout_session(request):
```

Comment:
Since we try to use Class Based Views could you move it into default VIew? Like here:
https://docs.djangoproject.com/en/4.2/topics/class-based-views/intro/#using-class-based-views

This would be nicely consistent with the rest of the site.

Reply:
Done.